### PR TITLE
Serial dilution update

### PR DESCRIPTION
**Pull request by john-c-lynch**
##Overview

Deleted get_values function at top causing errors

### CODE REVIEW

**Comment by john-c-lynch** [comment]
Same issue as before. make all wasn't run and the protobuilds folder causes problems then. Need to troubleshoot the make all on my local machine.

File: protocols/customizable_serial_dilution_ot2/customizable_serial_dilution.ot2.apiv2.py [hunk 1]
```diff
@@ -2,108 +2,123 @@
     'protocolName': 'Customizable Serial Dilution',
     'author': 'Opentrons <protocols@opentrons.com>',
     'source': 'Protocol Library',
-    'apiLevel': '2.5'
+    'apiLevel': '2.2'
```
Comment (chazmanc) [inline protocols/customizable_serial_dilution_ot2/customizable_serial_dilution.ot2.apiv2.py:5]: why lower this?

File: protocols/customizable_serial_dilution_ot2/customizable_serial_dilution.ot2.apiv2.py [hunk 2]
```diff
@@ -2,108 +2,123 @@
...
 
 def run(protocol_context):
-    [pipette_type, tip_type, trough_type, plate_type,
-     dilution_factor, num_of_dilutions, total_mixing_volume,
+    [pipette_type, dilution_factor, num_of_dilutions, total_mixing_volume,
         tip_use_strategy] = get_values(  # noqa: F821
-            'pipette_type', 'tip_type', 'trough_type', 'plate_type',
-            'dilution_factor', 'num_of_dilutions',
+            'pipette_type', 'dilution_factor', 'num_of_dilutions',
             'total_mixing_volume', 'tip_use_strategy'
```
Comment (chazmanc) [inline protocols/customizable_serial_dilution_ot2/customizable_serial_dilution.ot2.apiv2.py:13]: what happened to these?

**Review by chazmanc — CHANGES_REQUESTED** [review]
let's add a work in progress label to this PR (wip)

**Comment by john-c-lynch** [comment]
Made prior to make script working locally. See dilution_v2 branch for further work

**Pull request closed without merging**